### ISSUE TITLE
Clarification on license - Apache 2.0 or GPL 3.0 ?

### ISSUE THREAD

**Issue by sgustafsson** [typesense/typesense-java#69]
## Description

https://github.com/typesense/typesense-java?tab=readme-ov-file#license says:

> typesense-java is distributed under the Apache 2 license.

https://github.com/typesense/typesense-java?tab=readme-ov-file#installation says:

> The client is available on Maven central

On Maven central https://mvnrepository.com/artifact/org.typesense/typesense-java/0.9.0 , it says the license is GPL-3.0, as the pom.xml file says so:

```xml
<modelVersion>4.0.0</modelVersion>
<groupId>org.typesense</groupId>
<artifactId>typesense-java</artifactId>
<version>0.9.0</version>
<name>typesense-java</name>
<description>Typesense Java Client</description>
<url>https://github.com/typesense/typesense-java</url>
<licenses>
<license>
<name>GNU General Public License v3.0</name>
<url>https://www.gnu.org/licenses/gpl-3.0.en.html</url>
</license>
</licenses>
```

## Steps to reproduce

n.a.

## Expected Behavior

Consistent licensing information between GH repo and Maven Central.

## Actual Behavior

Inconsistent licensing information.

## Metadata

n.a.

**Typesense Version**: 0.9.0

**OS**: n.a.

**Comment by jasonbosco** [comment]
Thank you for catching that. typesense-java should be Apache 2.0 licensed. Only Typesense Server is GPL 3.0. 

We'll fix the license in Maven Central.

**Comment by sgustafsson** [comment]
Thanks for looking into this!

**Comment by kishorenc** [comment]
Fixed in `v1.0.0` release.
